src/Dfe.PlanTech.AzureFunctions/Functions/QueueReceiver.cs

```diff
@@ -17,12 +19,26 @@ public class QueueReceiver : BaseFunction
     private readonly CmsDbContext _db;
     private readonly ContentfulOptions _contentfulOptions;
     private readonly JsonToEntityMappers _mappers;
+    private readonly ServiceBusClient _serviceBusClient;
+    private readonly IConfiguration _configuration;
 
-    public QueueReceiver(ContentfulOptions contentfulOptions, ILoggerFactory loggerFactory, CmsDbContext db, JsonToEntityMappers mappers) : base(loggerFactory.CreateLogger<QueueReceiver>())
+    private const string CustomMessageProperty = "DeliveryAttempts";
+
+    private const int _defaultMaxMessageDeliveryAttempts = 4;
+    private const int _defaultMessageDeliveryDelayInSeconds = 10;
+
+    private readonly int _maxMessageDeliveryAttempts;
+    private readonly int _messageDeliveryDelayInSeconds;
+
+    public QueueReceiver(ContentfulOptions contentfulOptions, ILoggerFactory loggerFactory, CmsDbContext db, JsonToEntityMappers mappers, ServiceBusClient serviceBusClient, IConfiguration configuration) : base(loggerFactory.CreateLogger<QueueReceiver>())
     {
         _contentfulOptions = contentfulOptions;
         _db = db;
         _mappers = mappers;
+        _serviceBusClient = serviceBusClient;
+        _configuration = configuration;
+        _maxMessageDeliveryAttempts = int.TryParse(_configuration["MaxMessageDeliveryAttempts"], out var configuredMaxAttempts) ? configuredMaxAttempts : _defaultMaxMessageDeliveryAttempts;
```

Comment:
...and just use a local here as it's only used in the constructor and not elsewhere in the class